src/Components/ServiceScene/Breakdowns/LabelValuesBreakdownScene.tsx

```diff
@@ -31,13 +31,18 @@ import { DEFAULT_SORT_BY } from '../../../services/sorting';
 import { buildLabelsQuery, LABEL_BREAKDOWN_GRID_TEMPLATE_COLUMNS } from '../../../services/labels';
 import { getAppEvents } from '@grafana/runtime';
 import { getLabelGroupByVariable } from '../../../services/variableGetters';
-import { PanelMenu, getPanelWrapperStyles } from '../../Panels/PanelMenu';
+import { getPanelWrapperStyles, PanelMenu } from '../../Panels/PanelMenu';
+import { ClearFiltersLayoutScene } from './ClearFiltersLayoutScene';
+import { EmptyLayoutScene } from './EmptyLayoutScene';
+import { IndexScene } from '../../IndexScene/IndexScene';
+import { clearVariables, getVariablesThatCanBeCleared } from '../../../services/variableHelpers';
+import { ValueSummaryPanelScene } from './Panels/ValueSummary';
 
 type DisplayError = DataQueryError & { displayed: boolean };
 type DisplayErrors = Record<string, DisplayError>;
 
 export interface LabelValueBreakdownSceneState extends SceneObjectState {
-  body?: LayoutSwitcher & SceneObject;
+  body?: (LayoutSwitcher & SceneObject) | (ClearFiltersLayoutScene & SceneObject) | (EmptyLayoutScene & SceneObject);
```

Comment:
Is the `& SceneObject` necessary?

Reply:
Yeah this shit always throws me too, this is my current understanding:

The discriminated union of intersecting types `{(A extends C) & C} | {(B extends C) & C}` can only be replaced by a straight up discriminated union `{A extends C} | {B extends C}` if all types in B and A are common

This is the error you get if you try to define
```
body?: (LayoutSwitcher & SceneObject) | (SceneReactObject & SceneObject);
```
instead of
```
body?: (LayoutSwitcher) | (SceneReactObject);
```

Error:
```
TS2322: Type LayoutSwitcher | SceneReactObject<{}> is not assignable to type SceneReactObject<{}> & LayoutSwitcher
Type LayoutSwitcher is not assignable to type SceneReactObject<{}> & LayoutSwitcher
Type LayoutSwitcher is not assignable to type SceneReactObject<{}>
Types of property Component are incompatible.
Type SceneComponent<LayoutSwitcher> is not assignable to type SceneComponent<SceneReactObject<{}>>
Property onLayoutChange is missing in type SceneReactObject<{}> but required in type LayoutSwitcher
```

TL;DR Since `LayoutSwitcher` has `onLayoutChange` method, and `SceneReactObject` does not, `LayoutSwitcher |  SceneReactObject` would only contain the common properties of both.

^ Should only work if LayoutSwitcher and SceneReactObject had a common interface?